### PR TITLE
Spelling and lint fixes

### PR DESCRIPTION
**Pull request by jml**
Mostly just "cach" -> "catch". Also fixes some whitespace & unused imports while I was there.

[Review on Reviewable](https://reviewable.io/reviews/testing-cabal/fixtures/17)

### CODE REVIEW

File: fixtures/callmany.py [hunk 1]
```diff
@@ -96,5 +97,4 @@ def __enter__(self):
 
     def __exit__(self, exc_type, exc_val, exc_tb):
         self()
-        return False # propogate exceptions from the with body.
-
+        return False  # propogate exceptions from the with body.
```
Comment (elopio) [inline fixtures/callmany.py]: Shouldn't it be propagate? https://en.wiktionary.org/wiki/propogate

Comment (jml) [inline fixtures/callmany.py]: Fixed. Thanks.

File: fixtures/fixture.py [hunk 2]
```diff
@@ -157,7 +156,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
             self._cleanups()
         finally:
             self._remove_state()
-        return False # propogate exceptions from the with body.
+        return False  # propogate exceptions from the with body.
```
Comment (elopio) [inline fixtures/fixture.py]: Here is another propogate. I'm sorry I didn't mention it in the previous review.
The rest of the changes look good.

**Comment by thomir** [comment]
Looks good to me.


---







Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/testing-cabal/fixtures/17)

**Pull request merged**